tests/integration-tests/tests/networking/test_cluster_networking.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+import pytest
+from assertpy import assert_that
+
+
+@pytest.mark.dimensions("eu-west-1", "c4.xlarge", "alinux2", "slurm")
```

Comment:
let's use c5.xlarge only. I think we use c4 only in cn regions.

Reply:
I think eu-west-1 regions is heavily used already. Shall we pick a region with fewer tests? 

also we could select centos7 with one of the schedulers.